phpbench.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Is the addition of this file intentional?

Reply:
Nope.